docker-compose.f8ui.yml

```diff
@@ -6,6 +6,10 @@ services:
     image: fabric8-planner-platform
     # command: npm run test:unit
     environment:
+      FABRIC8_REALM: "fabric8-test"
+      FABRIC8_WIT_API_URL: "http://localhost:8080/api/"
+      FABRIC8_AUTH_API_URL: "http://localhost:8089/"
```

Comment:
This should be `http://localhost:8089/api`

Reply:
good catch, fix coming